src/backend/lib/upgrade-server.js

```diff
@@ -83,6 +83,7 @@ class UpgradeServer extends EventEmitter {
     this.stateLock.writeLock(release => {
       const self = this;
       function done(err) {
+        // Ignorable. this only fires if discovery.destroy() errors, which is fine
```

Comment:
If this error does not mean that the server is no longer is either the `IDLE` or `SEARCHING` state, then I don't think we should change the state here. What is the consequence of this error?

Reply:
> If this error does not mean that the server is no longer is either the IDLE or SEARCHING state, then I don't think we should change the state here.

I don't understand what you mean by this.

> What is the consequence of this error?

Unknown. Worst case, it's possible that the server is still advertising itself & responding to mDNS requests.